src/Conversation/DataClients/ScenarioDataClient.php

```diff
@@ -24,13 +34,171 @@ public function addFullScenarioGraph(Scenario $scenario): Scenario
+            Scenario::DESCRIPTION,
+            Scenario::INTERPRETER,
+            Scenario::CONDITIONS => Condition::FIELDS,
+            Scenario::BEHAVIORS => Behavior::FIELDS,
+            Scenario::STATUS,
+            Scenario::ACTIVE,
+            Scenario::CREATED_AT,
+            Scenario::UPDATED_AT,
+            Scenario::CONVERSATIONS => [
+                Conversation::OD_ID,
+                Conversation::NAME,
+                Conversation::DESCRIPTION,
+                Conversation::INTERPRETER,
+                Conversation::CONDITIONS => Condition::FIELDS,
+                Conversation::BEHAVIORS => Behavior::FIELDS,
+                Conversation::CREATED_AT,
+                Conversation::UPDATED_AT,
+                Conversation::SCENES => [
+                    Scene::OD_ID,
+                    Scene::NAME,
+                    Scene::DESCRIPTION,
+                    Scene::INTERPRETER,
+                    Scene::CONDITIONS => Condition::FIELDS,
+                    Scene::BEHAVIORS => Behavior::FIELDS,
+                    Scene::CREATED_AT,
+                    Scene::UPDATED_AT,
+                    Scene::TURNS => [
+                        Turn::OD_ID,
+                        Turn::NAME,
+                        Turn::DESCRIPTION,
+                        Turn::INTERPRETER,
+                        Turn::CONDITIONS => Condition::FIELDS,
+                        Turn::BEHAVIORS => Behavior::FIELDS,
+                        Turn::CREATED_AT,
+                        Turn::UPDATED_AT,
+                        Turn::RESPONSE_INTENTS => [
+                            Intent::OD_ID,
+                            Intent::NAME,
+                            Intent::DESCRIPTION,
+                            Intent::INTERPRETER,
+                            Intent::CONDITIONS => Condition::FIELDS,
+                            Intent::BEHAVIORS => Behavior::FIELDS,
+                            Intent::CREATED_AT,
+                            Intent::UPDATED_AT,
+                            Intent::SPEAKER,
+                            Intent::SAMPLE_UTTERANCE,
+                            Intent::CONFIDENCE,
+                            Intent::VIRTUAL_INTENT,
+                            Intent::MESSAGE_TEMPLATES => [
+                                MessageTemplate::UID,
+                                MessageTemplate::OD_ID,
+                                MessageTemplate::NAME,
+                                MessageTemplate::MESSAGE_MARKUP,
+                                MessageTemplate::DESCRIPTION,
+                                MessageTemplate::CREATED_AT,
+                                MessageTemplate::UPDATED_AT,
+                                MessageTemplate::CONDITIONS => [
+                                    Condition::OPERATION,
+                                    Condition::OPERATION_ATTRIBUTES,
+                                    Condition::PARAMETERS
+                                ],
```

Comment:
Very minor, but you should be able to replace these with the following (to avoid any potential divergence in future).

```suggestion
                                MessageTemplate::CONDITIONS => Condition::FIELDS,
```

Reply:
Thanks, updated